docs/user/translating.rst

```diff
@@ -375,34 +376,33 @@ bootstrapping translation for a new component using existing translations
 Rate limiting
 -------------
 
-To avoid abuse of the interface, there is rate limiting applied to several
-operations like searching, sending contact form or translating. In case you are
-hit by this, you are blocked for a certain period until you can perform the
+To avoid abuse of the interface, rate limiting is applied to several
+operations like searching, sending contact forms or translating. If affected by
+it, you are blocked for a certain period until you can perform the
 operation again.
 
 The default limits are described in the administrative manual in
-:ref:`rate-limit`, but can be tweaked by configuration.
+:ref:`rate-limit`, but can be fine-tuned in the Weblate configuration.
```

Comment:
see?

Reply:
The configuration is described on the rate limit link which is already present in this sentence.